test/ranch_ct_hook.erl

```diff
@@ -22,6 +22,17 @@ init(_, _) ->
 	%% ranch_sup children.
 	application:set_env(ranch, ranch_sup_intensity, 10),
 	application:set_env(ranch, ranch_sup_period, 1),
+	ok = application:load(ssl),
+	case {os:type(), application:get_key(ssl, vsn)} of
+		%% Internal active,N is broken on Windows since
+		%% OTP 21.2/ssl 9.1.
+		%% @todo Put an upper limit on the version when
+		%% this is fixed in a future OTP version.
+		{{win32, nt}, {ok, Vsn}} when Vsn >= "9.1" ->
```

Comment:
Problem is "10" >= "9.1" is false.

Reply:
I know, its done this way in other places (eg `acceptor_SUITE`) already, so did it like this here as well.

Maybe something like an application version decomposition/comparison function could be introduced into `ct_helper`, as this may be handy elsewhere (`cowboy`, maybe?) too? A problem is that the vsn can be practically anything, doesn't have to be "x.y.z", but for applications that are part of OTP this should work.

Reply:
Should have catched it earlier. We are getting dangerously close to 10. By the way some of the checks in acceptor_SUITE are probably no longer useful since we support 21+ now. In the meantime maybe match directly like `Vsn="9.3"++_`.

Reply:
I made a PR in `ct_helper` with a version decomposition function that may be of use here.